### PR TITLE
Vmware player improved2

### PR DESCRIPTION
**Pull request by bktan8**
The latest vmware player documentation.  Pls ignore the previous pull.

### CODE REVIEW

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 1]
```diff
@@ -1,116 +1,466 @@
...
+  .. code-block:: console
+
+    firmware = "efi"
+
+.. note::
+
+  VMware VM files are typically located in:
+
+  * Linux distros: `/home/username/vmware`
+  * Windows: `C:/\Users/\username/\Documents/\Virtual Machines`
```
Comment (tpleavitt) [inline source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: Also, in windows, this is a hidden file and in order to change it you need to unhide it

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 2]
```diff
@@ -1,116 +1,466 @@
...
+Install the VMware Workstation Player hypervisor
+================================================
 
-Player on Windows
------------------
+#. Enable the `Intel® Virtualization Technology`_ (Intel® VT) and the
+   `Intel® Virtualization Technology for Directed I/O`_ (Intel® VT-d) in the
+   host machine’s BIOS.
+#.  VMware Workstation 14 Player is available for Windows and Linux.  Download 
+    the preferred version of choice.
```
[the remark this answers is not in the document]
Comment (bktan8) [reply source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: Done.

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 3]
```diff
@@ -1,116 +1,466 @@
...
+    * Give it a name.
+    * Click :guilabel:`Next`.
+
+      |vmware-player-04|
+
+      Figure 4: VMware Workstation 14 Player - Name virtual machine
+
+#.  :guilabel:`Specify Disk Capacity`:
+
+    * Set the desired disk size.
```
[the remark this answers is not in the document]
Comment (bktan8) [reply source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: I added a reference to the "Mininum System Requirements" doc.

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 4]
```diff
@@ -1,116 +1,466 @@
...
+      Figure 6: VMware Workstation 14 Player - Customize hardware
+
+    * Select :guilabel:`Processors` > :guilabel:`Virtualization engine`, and 
+      check :guilabel:`Virtualize Intel VT-x/EPT or AMD-V/RVI`.
+
+      |vmware-player-07|
+
+      Figure 7: VMware Workstation 14 Player - Set virtualization engine option
+
+    * Click :guilabel:`Close`.
```
[the remark this answers is not in the document]
Comment (bktan8) [reply source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: Ok - I recommended 2GB for just the installer ISO to function properly.  I also stated that once the installation is complete, the memory can be dialed down -- if desired.  also provided a reference to "Minimum System Requirements" doc.

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst (hunk 1 above)
Comment (bktan8) [reply source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: I verified on my Windows system that it is set to "Don't show hidden files, folders, or drives".  When I browsed to the "Virtual Machines" folder via Explorer, I can see the .vmx file.  Its type is "VMware virtual machine configuration".  I added this extra bit of information into the doc.

Comment (tpleavitt) [reply source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: The problem I had was when I edited the file and tried to save it after adding firmware = "efi", it wouldn't let me save it directly.

Comment (tpleavitt) [reply source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: Another issue I had was when I tried to add the vmware extra software, I couldn't figure out what the cdrom /dev path was and couldn't mount it to add the additional software.  As far as I know I may not be able to add the SW at all to clear linux.

**Comment by bktan8** [comment]
@rcaballeromx - commits SQUASHED and figures substitutions converted to "inline".  Ready for review.

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 5]
```diff
@@ -1,116 +1,528 @@
...
-<http://www.intel.com/content/www/us/en/virtualization/virtualization-technology/intel-virtualization-technology.html>`_
-(Intel® VT) and `Intel® Virtualization Technology for Directed I/O
-<https://software.intel.com/en-us/articles/intel-virtualization-technology-for-directed-io-vt-d-enhancing-intel-platforms-for-efficient-virtualization-of-io-devices>`_
-(Intel® VT-d) in your BIOS/UEFI firmware configuration.
+This section explains how to run |CLOSIA| as a virtual machine using the 
+VMware Workstation 14 Player hypervisor.
 
-Install VMware Player
-=====================
+There are 2 ways to create a |CL| :abbr:`VM (Virtual Machine)` to run in VMware:
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: Incorrect verb conjugation:  There are two ways to create a |CL| :abbr:`VM (Virtual Machine)` running in VMware:

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 6]
```diff
@@ -1,116 +1,528 @@
...
 
-VMware Workstation Player, formerly VMware Player, is a virtualization
-software package for x64 computers running Microsoft Windows or Linux.
-Download VMware player from the `VMware website`_.
+* `Method #1`: Install |CL| into a new VM.  This provides flexibility
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: Do not use back-ticks for emphasis. Use either \*\***bold**\*\* or \**italics*\*

Comment (rcaballeromx) [reply source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: Avoid relative pronouns such as "This". This issue can be easily avoided by adding the correct noun. "This method"

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 7]
```diff
@@ -1,116 +1,528 @@
...
 
-Install VMware Player
-=====================
+There are 2 ways to create a |CL| :abbr:`VM (Virtual Machine)` to run in VMware:
 
-VMware Workstation Player, formerly VMware Player, is a virtualization
-software package for x64 computers running Microsoft Windows or Linux.
-Download VMware player from the `VMware website`_.
+* `Method #1`: Install |CL| into a new VM.  This provides flexibility 
+  in configuring the VM size, partitions, initial |CL| bundles selection, etc.
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: Confusing text: ... "in configuring the VM, for example: its size, the number of partitions, the initial selection of bundles, etc."

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 8]
```diff
@@ -1,116 +1,528 @@
...
+* `Method #2`: Use a ready-made VMware |CL| image.
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: Confusing: replace ready-made with pre-configured or supplied and link to the downloads page.

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 9]
```diff
@@ -1,116 +1,528 @@
...
-VMware Workstation Player, formerly VMware Player, is a virtualization
-software package for x64 computers running Microsoft Windows or Linux.
-Download VMware player from the `VMware website`_.
+* `Method #1`: Install |CL| into a new VM.  This provides flexibility 
+  in configuring the VM size, partitions, initial |CL| bundles selection, etc.
+* `Method #2`: Use a ready-made VMware |CL| image.  
 
-Player on Linux
----------------
+Both are discussed below.
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: Both what? Methods, versions of VMWare, images?

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 10]
```diff
@@ -1,116 +1,528 @@
...
-Player on Linux
----------------
+Both are discussed below.  
 
-For the Linux option, you should have a :file:`VMware-Player-{version}_FILE.bundle`
-file. To install it, run:
+.. note::
 
-::
+  The figures shown throughout this document are from the Windows\* version of
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: Passive voice, clarity, and simplicity: "We captured the figures used in this tutorial using VMware Workstation 14 Player for Windows\*. The menus and prompts are similar to the Linux version save for minor wording differences.

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 11]
```diff
@@ -1,116 +1,528 @@
...
-  $ sudo bash ./VMware-Player-{version}.x86_64.bundle
+Install the VMware Workstation Player hypervisor
+================================================
 
-Player on Windows
------------------
+#. Enable the `Intel® Virtualization Technology`_ (Intel® VT) and the
+   `Intel® Virtualization Technology for Directed I/O`_ (Intel® VT-d) in the
+   host machine’s BIOS.
+#.  `VMware Workstation 14 Player`_ is available for Windows and Linux.  Download
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: Consistency: Download the `VMware Workstation 14 Player`_ for either Windows or Linux as you prefer.

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 12]
```diff
@@ -1,116 +1,528 @@
...
------------------
+#. Enable the `Intel® Virtualization Technology`_ (Intel® VT) and the
+   `Intel® Virtualization Technology for Directed I/O`_ (Intel® VT-d) in the
+   host machine’s BIOS.
+#.  `VMware Workstation 14 Player`_ is available for Windows and Linux.  Download 
+    the preferred version of choice.
+#.  Install it.
 
-Follow the instructions from the Setup Assistant.
+    * For Linux distros supported by VMware:
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: Do not use lists to split instructions for multiple OSes. Instead create two separate sections one for each OS. This comment applies throughout the tutorial.

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 13]
```diff
@@ -1,116 +1,528 @@
...
+#.  `VMware Workstation 14 Player`_ is available for Windows and Linux.  Download 
+    the preferred version of choice.
+#.  Install it.
 
-Follow the instructions from the Setup Assistant.
+    * For Linux distros supported by VMware: 
 
+      #.  Enable a GUI desktop.  
+      #.  Start a terminal emulator.
+      #.  Start the installer and follow the guided steps.
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: #. Start the installer and follow the provided steps.

File: source/clear-linux/get-started/virtual-machine-install/vmware-player.rst [hunk 14]
```diff
@@ -1,116 +1,528 @@
...
 
-Prepare Image
-=============
+        .. code-block:: console
 
-#. Download the `latest`_ |CL| **live** version (clear-XXXX-live.img.xz)
+          $ sudo sh ./VMware-Player-<version>.x86_64.bundle
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/virtual-machine-install/vmware-player.rst]: If this is the command to start the installer, which I know it is, there is no indication of that on the steps above.

**Comment by bktan8** [comment]
@rcaballeromx - Ready for your review.

File: source/clear-linux/get-started/virtual-machine-install/vmware-player-install-cl.rst [hunk 15]
```diff
@@ -0,0 +1,301 @@
...
+    firmware = "efi"
+
+VMware typically saves VM files on the following locations:
+
+* On Linux distros: :file:`/home/username/vmware`
+* On Windows: :file:`C:/\Users/\username/\Documents/\Virtual Machines`
+
+The file type is `VMware virtual machine configuration`.
+
+Power on the VM
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/virtual-machine-install/vmware-player-install-cl.rst]: @bktan8 I expected this section to include the steps to power on the VM. Can you add those steps or are they included somewhere else?

**Comment by rcaballeromx** [comment]
@bktan8 I'm done with edits and now I am going to fix the file names for both the rst files and the figures.
Meanwhile, could you take a look into adding the instructions to power on the VMs in both cases: with pre-configured image and without?

**Comment by bktan8** [comment]
@rcaballeromx - added details for the "power on" section for both docs.  

Also, in the "preconf" doc -> Create and configure -> step 10, I changed "When you attach the pre-configured |CL| VMware image, VMWare removes the default virtual disk making this setting irrelevant" to "The disk settings do not matter at this stage because you will remove the default virtual disk and attach a pre-configured Clear Linux VMware image at a later step" because the former implies that VMware will do it automatically, but in fact, the user will manually remove the default virtual disk.

**Comment by rcaballeromx** [comment]
@bktan8 I'll take a look at the new sections. In the future, please comment in-line in the file. Commenting on specific changes on a general comment is not considered a good practice.

File: source/clear-linux/get-started/virtual-machine-install/vmw-player-preconf.rst [hunk 16]
```diff
@@ -0,0 +1,314 @@
...
+   .. figure:: figures/vmw-player-preconf/vmw-player-preconf-04.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Name virtual machine
+
+      Figure 4: VMware Workstation 14 Player - Name virtual machine
+
+#. Click the :guilabel:`Next` button.
+
+#. On the :guilabel:`Specify Disk Capacity` screen, click the :guilabel:`Next`
+   button. The disk settings do not matter at this stage because you will remove
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/virtual-machine-install/vmw-player-preconf.rst]: The wording added is too complex and references future steps without specificity which causes confusion in readers. Both things were avoided in the previous version. I suggest the following alternative:
Keep the default disk settings unchanged. When we attach the pre-configured |CL| VMware image, VMWare removes the default virtual disk with the pre-configured disk.

File: source/clear-linux/get-started/virtual-machine-install/vmw-player.rst [hunk 17]
```diff
@@ -0,0 +1,313 @@
...
+
+   .. figure:: figures/vmw-player/vmw-player-13.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Power on virtual machine
+
+      Figure 13: VMware Workstation 14 Player - Power on virtual machine
+
+#. Click :guilabel:`Play virtual machine`.
+
+Also see:
```
Comment (rcaballeromx) [inline source/clear-linux/get-started/virtual-machine-install/vmw-player.rst]: For consistency and parallelism I would change this line to:
For other guides on using the VMWare player, see:

**Pull request merged**